### ISSUE TITLE
It's broken!

### ISSUE THREAD

**Issue by yoyostile** [pwaldhauer/amazon-account-crawler#14]
![download](https://cloud.githubusercontent.com/assets/573120/10495714/7d6e8aa4-72bd-11e5-873c-0e884e724f4f.jpg)

```
/Users/yoyostile/.rvm/gems/ruby-2.2.0/gems/mechanize-2.7.3/lib/mechanize.rb:354:in `click': undefined method `[]' for nil:NilClass (NoMethodError)
        from amazon.rb:83:in `block in <main>'
        from /Users/yoyostile/.rvm/gems/ruby-2.2.0/gems/mechanize-2.7.3/lib/mechanize.rb:442:in `get'
        from amazon.rb:71:in `<main>'
```

Maybe I'll fix this. Let's see what happens.

**Comment by Sillium** [comment]
+1

**Comment by slcr** [comment]
+1

**Comment by pwaldhauer** [comment]
+1

**Comment by pwaldhauer** [comment]
I think I'll fix is over the holidays, because it's the end of the year and I need to write my yearly blog post!

**Comment by yoyostile** [comment]
Hey, holidays are coming. I hope I'll have some spare time for this.

**Comment by yoyostile** [comment]
Ah great, @pwaldhauer was a tad faster. :+1:

**Comment by pwaldhauer** [comment]
Okay, I tried to fix this quickly but the error seems to be somewhere deep inside... My motivation to fix this is not really present at the moment. Maybe someone should reimplement this with phantomjs or so...

**Comment by pwaldhauer** [comment]
Fixed by #15
